docs/04-operation-guides/troubleshooting/evnt-02-jetstream-troubleshooting.md

```diff
@@ -12,30 +12,31 @@ Events were not received by the consumers.
 
 2. Use the [nats CLI](https://github.com/nats-io/natscli) to check if the stream was created:
 
-    1. Port forward the Kyma Eventing NATS Service to localhost. Use port `4222`. Run:
-        ```bash
-        kubectl -n kyma-system port-forward svc/eventing-nats 4222
-        ```
-    2. Use the [nats CLI](https://github.com/nats-io/natscli) to list the streams:
-       ```bash
-        $ nats stream ls
-        ╭────────────────────────────────────────────────────────────────────────────╮
-        │                                  Streams                                   │
-        ├──────┬─────────────┬─────────────────────┬──────────┬───────┬──────────────┤
-        │ Name │ Description │ Created             │ Messages │ Size  │ Last Message │
-        ├──────┼─────────────┼─────────────────────┼──────────┼───────┼──────────────┤
-        │ sap  │             │ 2022-05-03 00:00:00 │ 0        │ 318 B │ 5.80s        │
-        ╰──────┴─────────────┴─────────────────────┴──────────┴───────┴──────────────╯
-        ```       
-
-    3. If the stream exists, check the timestamp of the `Last Message` that the stream received. A recent timestamp would mean that the event was published correctly.
+       1. Port forward the Kyma Eventing NATS Service to localhost. Use port `4222`. Run:
```

Comment:
For lines 15-31, make sure that only code is in the code block. The text should be outside of it.